tests/integration_tests.py

```diff
@@ -0,0 +1,174 @@
+import json
+import requests
+
+base_url = "http://localhost:8080/ga4gh/drs/v1"
+data_objects_path = "data_objects.json"
+headers = {
+    'accept': 'application/json',
+    'Content-Type': 'application/json'
+}
+payload_objects = json.dumps({
```

Comment:
Can some object from "data_objects.json" perhaps be reused here instead of hardcoding another 50 lines of payload in here?